src/riak_kv_eleveldb_backend.erl

```diff
@@ -49,6 +49,7 @@
 
 -export([data_size/1]).
 
+-compile([export_all]).
```

Comment:
There is no explanation I can give for it to be there. It just emerged from the conflict resolution pains. I have removed it.